config/config.yaml

```diff
@@ -0,0 +1,2 @@
+ENV_PRODUCT: rke2
```

Comment:
on the current master this is already ignored on git to facilitate our lifes not needing to add this everytime on PRs